library/leakcanary-analyzer/src/main/java/com/squareup/leakcanary/HeapAnalyzer.java

```diff
@@ -62,7 +62,7 @@
   private final ExcludedRefs excludedRefs;
 
   public HeapAnalyzer(ExcludedRefs excludedRefs) {
-    this(new ExcludedRefs(), excludedRefs);
+    this(new ExcludedRefs.Builder().build(), excludedRefs);
```

Comment:
There are two usages of empty ExcludedRefs. Maybe define it ExcludedRefs.EMPTY or something like?